### Goal (arguments) definition
# terrain to traverse
convex_plane_decomposition_msgs/PlanarTerrain terrain
# start and goal configurations
SingleRigidBody start_state
SingleRigidBody goal_state
---
### Result (return values) definition
# trajectory containing the generated trajectory at each timestep
SplineHolder trajectory
# The contact_data for each foot at each timestep
ContactDatum[] contact_data
---
### Feedback
# int32[] sequence